admin/run.mjs

```diff
@@ -0,0 +1,120 @@
+/* eslint-disable no-console */
+import 'regenerator-runtime/runtime';
+
+import EsInstance from '../dist/src/ElasticSearchClientInstance';
+import { countNOfDocs, createIndexIfNeeded } from '../dist/src/esUtils';
+
+import { ArrangerApi } from './arrangerApi.mjs';
+import { projectsConfig } from './projectsConfig.mjs';
+import { esHost } from '../dist/src/env.js';
+
+const hasProjectArrangerMetadataIndex = async (esClient, projectName) => {
+    const r = await esClient.indices.exists({
+        index: ArrangerApi.getProjectMetadataEsLocation(projectName).index,
+    });
+    return !!r?.body;
+};
+
+const isProjectListedInArranger = async (esClient, projectName) => {
+    const r = await esClient.exists({
+        index: ArrangerApi.constants.ARRANGER_PROJECT_INDEX,
+        id: projectName,
+    });
+    return !!r?.body;
+};
+
+const sameIndices = (xs, ys) => {
+    const s = new Set(ys);
+    return xs.length === ys.length && xs.every(x => s.has(x));
+};
+
+//===== Start =====//
+console.info(`admin-project-script - Starting script`);
+
+//values are hardcoded for now, but as soon as possible, we should use env var from env.ts
```

Comment:
Ah ok. i see this here... why its not possible now?

Reply:
I don`t want to create environment variables since we are kinda in a twilight zone. When indices will be fully migrated à la include sauce I will make things clean.